lib/hosting_environment.rb

```diff
@@ -15,4 +15,29 @@ def self.phase(current_service)
                  env
                end
   end
+
+  def self.dfe_sign_in_client_id(current_service)
+    ENV["#{current_service.upcase}_DFE_SIGN_IN_CLIENT_ID"]
```

Comment:
I'm worried this will be hard to pin down if something goes wrong here. Can we use a hash lookup like above instead of string manipulation?

Reply:
Addressed https://github.com/DFE-Digital/itt-mentor-services/pull/184/commits/40e0a910922358b89dc9c8585fbfb06bb82a1ec5